app/mailers/app_mailer.rb

```diff
@@ -1,7 +1,11 @@
-class AppMailer < ActionMailer
+class AppMailer < ActionMailer::Base
   def welcome_user_upon_registration(user)
     @user = user
-    mail from: 'info@myflix.com', to: user.email, subject: 'Welcome to myflix!'
+    mail from: 'info@myflix.com', to: user.email, subject: 'Welcome to Myflix!'
```

Comment:
You can refactor `from: 'info@myflix.com'` to `default from: 'info@myflix.com'` or something similar, it works the same way as `before_action` in Controllers.